src/Cms/Content.php

```diff
@@ -256,7 +251,8 @@ public function toArray(): array
      */
     public function update(array $content = null, bool $overwrite = false)
     {
-        $this->data = $overwrite === true ? (array)$content : array_merge($this->data, (array)$content);
+        $content = array_change_key_case((array)$content);
```

Comment:
Does it need the `(array)` casting here? Wouldn't it be easier to turn the parameter then into `array $content = []`?

Reply:
That would be a breaking-change as you could previously explicitly pass `null`. I wonder why we allowed `null` in the first place as it looks like a pretty deliberate and explicit feature.

@bastianallgeier Do you remember?

Reply:
I have no idea why the default is null. It makes no sense while looking at it. Do we break any of our tests when we change it? I really wouldn't mind going for the more useful default here.

Reply:
Maybe it was another case of dealing with weird user input (e.g. from `get()`). But in this case I think it's pretty unlikely that anyone might have used it. It would still be a breaking change.

If I remember correctly, our tests would still pass, but of course that's not a guarantee. What do you think: Should we risk it?

Reply:
Would it be a big deal just to keep it as it is to avoid yet another breaking change?

Reply:
We keep stacking them up all over the place. On the other hand it's another "a lot of pain once vs. smaller portions of pains in the next releases" kind of situation.

Reply:
I'd ask ourselves how much this change helps us? How much delaying this change until 3.7 would block us in other aspects? And on that basis decide. If it helps us to move forward on other aspects and features, let's already include it. If it's unrelated and should just be changed eventually, maybe it can wait until 3.7